code/game/turfs/open/floor/light_floor.dm

```diff
@@ -87,7 +89,15 @@
 
 	switch(state)
 		if(LIGHTFLOOR_FINE)
-			icon_state = "light_on-[LAZYFIND(coloredlights, currentcolor)]"
+			if(!cycle)
```

Comment:
Invert this, as in change this to `if (cycle)` and copy the else.

Reply:
Done